Tempoross/alert.json

```diff
@@ -0,0 +1,9 @@
+{
+  "displayName": "Tempoross",
+  "description": "Alerts for Game Starting, Incoming Wave, Vulnerable, Clouds, Game Over. Uses ElevenLabs voices if you have a api key set.",
+  "compatibleVersion": "3.0.0",
+  "author": "Mr_Pryor",
+  "category": "COMBAT",
+  "tags": [ "minigame", "Tempoross" ],
+  "alert": [{"type":"ChatAlert","message":"The Spirit Anglers weigh anchor and sail you out to face Tempoross.","regexEnabled":false,"enabled":true,"name":"Game Starting","debounceTime":0,"notifications":[{"type":"TextToSpeech","gain":5,"rate":1,"voice":"GEORGE","source":"ELEVEN_LABS","elevenLabsVoiceId":"Yko7PKHZNXotIFUBG7I9","message":"Sailing out!","fireWhenFocused":true}]},{"type":"ChatAlert","message":"A colossal wave closes in...","regexEnabled":false,"enabled":true,"name":"Wave","debounceTime":0,"notifications":[{"type":"TextToSpeech","gain":5,"rate":1,"voice":"GEORGE","source":"ELEVEN_LABS","elevenLabsVoiceId":"Yko7PKHZNXotIFUBG7I9","message":"A colossal wave closes in...","fireWhenFocused":true}]},{"type":"ChatAlert","message":"Tempoross is vulnerable!","regexEnabled":false,"enabled":true,"name":"Vulnerable","debounceTime":0,"notifications":[{"type":"TextToSpeech","gain":5,"rate":1,"voice":"GEORGE","source":"ELEVEN_LABS","elevenLabsVoiceId":"Yko7PKHZNXotIFUBG7I9","message":"Tempoross is vulnerable!","fireWhenFocused":true}]},{"type":"ChatAlert","message":"A charged torrent of water surges towards a nearby cannon...","regexEnabled":false,"enabled":true,"name":" Torrent","debounceTime":0,"notifications":[{"type":"TextToSpeech","gain":5,"rate":1,"voice":"GEORGE","source":"ELEVEN_LABS","elevenLabsVoiceId":"Yko7PKHZNXotIFUBG7I9","message":"Water surge!","fireWhenFocused":true}]},{"type":"ChatAlert","message":"A strong wind blows as clouds roll in...","regexEnabled":false,"enabled":true,"name":"Clouds","debounceTime":0,"notifications":[{"type":"TextToSpeech","gain":5,"rate":1,"voice":"GEORGE","source":"ELEVEN_LABS","elevenLabsVoiceId":"Yko7PKHZNXotIFUBG7I9","message":"Clouds roll in.","fireWhenFocused":true}]},{"type":"ChatAlert","message":"The skies clear as Tempoross retreats to the depths.","regexEnabled":false,"enabled":true,"name":"Game Over","debounceTime":0,"notifications":[{"type":"TextToSpeech","gain":5,"rate":1,"voice":"GEORGE","source":"ELEVEN_LABS","elevenLabsVoiceId":"Yko7PKHZNXotIFUBG7I9","message":"Tempoross retreats.","fireWhenFocused":true}]}]
```

Comment:
So I have the format working a little differently here. `alert` must be a single `AlertGroup`. I have yet to update the exporting to reflect that so it still exports as an array.

Reply:
This is technically subject to change, as I haven't decided how I want to change the exporting from the UI.